components/channel_header_dropdown/channel_header_dropdown_items.tsx

```diff
@@ -11,7 +11,7 @@ import {isGuest} from 'mattermost-redux/utils/user_utils';
 
 import MobileChannelHeaderPlug from 'plugins/mobile_channel_header_plug';
 
-import ChannelMoveToSubMenu from 'components/channel_move_to_sub_menu';
+import ChannelMoveToSubMenuOld from 'components/channel_move_to_sub_menu_old';
```

Comment:
If I am not wrong, I think it's industry practice to rename things with the Legacy prefix or suffix in a scenario like this so maybe `ChannelMoveToSubMenuLegacy`. No need to change anything here.